pandas/conftest.py

```diff
@@ -640,6 +640,24 @@ def any_int_dtype(request):
     return request.param
 
 
+@pytest.fixture(params=ALL_EA_INT_DTYPES)
```

Comment:
any_Int_dtype would be fine.

Reply:
Maybe `any_nullable_int_dtype` to avoid caps and be a bit more explicit?

Reply:
yes that's fine